totalRP3/modules/register/msp/register_msp.lua

```diff
@@ -150,7 +150,7 @@ local function onStart()
 		for i = 1, 5 do
 			local peek = dataTab.PE[tostring(i)];
 			if peek and peek.AC then
-				if i > 1 then
+				if #peeks > 1 then

```

Comment:
Should be > 0

Reply:
Rebased with this fixed. I actually made that error multiple times, including when we were talking about it in Discord.